packages/catalog-platformstate-writer/src/index.ts

```diff
@@ -0,0 +1,44 @@
+import { EachMessagePayload } from "kafkajs";
+import { logger, decodeKafkaMessage } from "pagopa-interop-commons";
+import { runConsumer } from "kafka-iam-auth";
+import { EServiceEvent } from "pagopa-interop-models";
+import { match } from "ts-pattern";
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import { handleMessageV1 } from "./consumerServiceV1.js";
+import { handleMessageV2 } from "./consumerServiceV2.js";
+import { config } from "./config/config.js";
+
+const dynamoDBClient = new DynamoDBClient({
```

Comment:
I think we can derive its config from aws.local like we already do [here](https://github.com/pagopa/interop-be-monorepo/blob/03e45af728888bc9fad8084d485154ba9d447c3c/packages/backend-for-frontend/src/routers/privacyNoticeRouter.ts#L30)

Reply:
I agree, but this would complicate the test setup. The endpoint property has to be set after the testcontainer is initialized (we don't know the mapped port earlier). So if we remove this property from the config, where do we store [that](https://github.com/pagopa/interop-be-monorepo/pull/937/files#diff-5ab26860a8d06d3d8dc575a59fecf4245424952ff297c97493696dadf91212d9R143-R144) value?

Reply:
We could follow the approach we used for the `ReadModel`, we still pass the configs but using a function in [commons](https://github.com/pagopa/interop-be-monorepo/blob/9e58fc5105b01fb7e8e2aab680e2bcb4d9ee768c/packages/commons/src/repositories/ReadModelRepository.ts#L201)

Reply:
After a brief brainstorming we decided to postpone this improvement that will be tracked in [IMN-841](https://pagopa.atlassian.net/browse/IMN-841)